client/van_connector_create_test.go

```diff
@@ -61,41 +80,45 @@ func TestConnectorCreateInterior(t *testing.T) {
 	testPath := "./tmp/"
 	os.Mkdir(testPath, 0755)
 
-	for _, c := range testcases {
-		ctx, cancel := context.WithCancel(context.Background())
-		defer cancel()
+	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
 
-		secretsFound := []string{}
+	//cli, err := NewMockClient("skupper", "", "")
+	secrets := make(chan *corev1.Secret, 10) //TODO why 10?
+	informers := informers.NewSharedInformerFactory(cli.KubeClient, 0)
+	secretsInformer := informers.Core().V1().Secrets().Informer()
+	secretsInformer.AddEventHandler(&cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
+			secret := obj.(*corev1.Secret)
+			t.Logf("secret! added: %s/%s", secret.Namespace, secret.Name)
+			//if !strings.HasPrefix(secret.Name, "skupper") {
+			if strings.HasPrefix(secret.Name, "conn") { //just changed to make the example work
+				t.Logf("meet condition! \n")
+				secrets <- secret
```

Comment:
This change is not needed for the "integration" demo, it is just there because is part of this draft pr being pushed.